shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/AlertSignificance.kt

```diff
@@ -1,19 +1,22 @@
 package com.mbta.tid.mbta_app.model
 
-/**
- * | [AlertSignificance]     | [Major] | [Secondary] | [Accessibility] | [Minor] | [None] |
- * |-------------------------|---------|-------------|-----------------|---------|--------|
- * | Shown on map            | Yes     | No          | No              | No      | No     |
- * | Replaces upcoming trips | Yes     | No          | No              | No      | No     |
- * | Shown in nearby transit | Yes     | Yes         | Yes             | No      | No     |
- * | Shown in stop details   | Yes     | Yes         | Yes             | Yes     | No     |
- * | Shown in trip details   | Yes     | No          | No              | No      | No     |
- */
```

Comment:
I don't think this table is all that clear - trip details is halfway to not existing anymore - so I figured now was a good time to swap in some more specific documentation on each value separately.